internal/oci/container.go

```diff
@@ -413,10 +414,10 @@ func (c *Container) Pid() (int, error) {
 // and it is the same process that was originally started by the runtime.
 func (c *Container) pid() (int, error) {
 	if c.state == nil {
-		return 0, errors.New("state not initialized")
+		return 0, ErrNotInitialized
 	}
 	if c.state.InitPid <= 0 {
-		return 0, errors.New("PID not initialized")
+		return 0, ErrNotInitialized
```

Comment:
Do we really want to have the same error message being spun up for both errors?

Reply:
yeah the above check is mostly a sanity check to prevent us from segfaulting. The result is the same for both: we're not in a position to check the pid yet